website/docs/r/linux_virtual_machine_scale_set.html.markdown

```diff
@@ -196,6 +196,18 @@ resource "azurerm_linux_virtual_machine_scale_set" "example" {
 
 * `proximity_placement_group_id` - (Optional) The ID of the Proximity Placement Group in which the Virtual Machine Scale Set should be assigned to. Changing this forces a new resource to be created.
 
+* `resilient_vm_creation_enabled` - (Optional) Should resilient VM creation be enabled? When enabled, the service will attempt to create VMs in alternative fault domains or zones if the primary location fails during creation. Defaults to `false`.
+
+-> **Note:** `resilient_vm_creation_enabled` is currently not supported in the `austriaeast`, `belgiumcentral`, `centraluseuap`, `chilecentral`, `indonesiacentral`, `israelnorthwest`, `malaysiawest`, `mexicocentral`, `newzealandnorth`, `southcentralus2`, `southindia`, `southeastus3`, `southwestus`, `eastasia`, `eastus`, `southcentralus`, `southeastasia`, and `westeurope` regions.
+
+~> **Note:** Removing the `resilient_vm_creation_enabled` field from the configuration after it has been set to `true` will force a new resource to be created.
+
+!> **Note:** Azure does not support disabling resiliency policies. Once the `resilient_vm_creation_enabled` field is set to `true`, it cannot be reverted to `false`.
```

Comment:
Maybe the API is reporting a false positive, but I was able to make a VMSS and toggle both creation and deletion resiliency policies on and off repeatedly without errors, and with the subsequent GET requests reflecting the updated value. Can you include a link to the spec or documentation that talked about this limitation?